### PR TITLE
Add Eclipse Public License 2.0

### PR DESCRIPTION
**Pull request by geoand**
Fixes: SB-1414

### CODE REVIEW

File: etc/rh-license-names.json [hunk 1]
```diff
@@ -203,6 +203,20 @@
...
     ]
   },
+  {
+    "name": "Eclipse Public License, Version 2.0",
+    "url": "http://www.eclipse.org/legal/epl-v20.html",
+    "aliases": [
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License 2.0",
+      "Eclipse Public License v2.0",
+      "Eclipse Public License - v 2.0",
```
Comment (gytis) [inline etc/rh-license-names.json]: "Eclipse Public License - v 2.0" should go to the "name" and the "Eclipse Public License, Version 2.0" should go here. The idea of this override is whenever any of these aliases is found to replace them with the value in "name". Values in "name" and "url" have to be found in http://git.app.eng.bos.redhat.com/git/jboss-licenses.git/plain/rh_license_list.json

Comment (geoand) [reply etc/rh-license-names.json]: Great, thanks! How about now?

File: etc/rh-license-names.json [hunk 2]
```diff
@@ -203,6 +203,20 @@
...
+    "url": "http://www.eclipse.org/legal/epl-v20.html",
+    "aliases": [
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License 2.0",
+      "Eclipse Public License v2.0",
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License (EPL), Version 2.0"
+    ],
+    "urlAliases": [
+      "http://www.eclipse.org/legal/epl-v20.html"
```
Comment (gytis) [inline etc/rh-license-names.json]: Just to be on a safe side in the future could also add http://www.eclipse.org/legal/epl-2.0 to urlAliases. It's the one that was used in the dependency that caused a QE failure.

Comment (gytis) [reply etc/rh-license-names.json]: Sorry just noticed it now

**Comment by gytis** [comment]
Everything else is great.

File: etc/rh-license-names.json (hunk 2 above)
Comment (geoand) [reply etc/rh-license-names.json]: Coolio, fixed :)

**Pull request merged**